interfaces/src/db.ts

```diff
@@ -160,6 +165,8 @@ interface Patches {
   patches: Patch[];
 }
 
+export type KeyedVersions = Array<[string, Version]>;
```

Comment:
But `type` is only an alias, not a type (so like `typedef` in C and especially in C++, and not at all like `type` in Go).  So we'll have confusing error messages regarding this type, sometimes it will be `Key` and sometimes it will be `string`.  Given that `Key` is such a useful name, I'd rather not.